### PR TITLE
Serverless integration cleanup

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- remove old helm tests
- remove test image build 

**Related issue(s)**
https://github.com/kyma-project/serverless-manager/issues/44

### CODE REVIEW

File: templates/data/kyma-integration-data.yaml [hunk 1]
```diff
@@ -471,29 +464,7 @@ templates:
...
-                    - "extra_refs_test-infra"
-                  local:
-                    - "jobConfig_default"
-                    - "request_small"
-                    - "vm_job_template_serverless_k3s"
-                    - "vm_job_labels_template"
-              - jobConfig:
-                  name: "pre-main-serverless-integration-k3s-local"
-                  # following regexp won't start build if only Markdown files were changed
                   run_if_changed: "^((tests/function-controller\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))"
```
Comment (dbadura) [inline templates/data/kyma-integration-data.yaml:467]: Add `resources/serverless\` directory

**Review by pPrecel — DISMISSED** [review]
LGTM

**Review by pPrecel — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**